server/schemas/enquiry.schema.js

```diff
@@ -23,7 +24,7 @@ export const addEnquirySchema = Joi.object({
   occupation: requiredString('Occupation'),
   email: requiredEmail('Email Address'),
   nameOnTitleDocument: requiredString('Name on Title Document'),
-  investmentFrequency: requiredString('Investment Frequency'),
+  investmentFrequency: requiredPaymentFrequency('Investment Frequency'),
```

Comment:
Lets make this requiredNumber

```suggestion
  investmentFrequency: requiredNumber('Investment Frequency'),
```